### PR TITLE
feat(kbreadcrumbs): component reskin [KHCP-8969]

### PR DESCRIPTION
**Pull request by portikM**
# Summary

[Jira](https://konghq.atlassian.net/browse/KHCP-8969)
[Figma](https://www.figma.com/file/Yze0SWXl5nKjR0rFdilljK/Components?type=design&node-id=2020%3A9543&mode=dev&t=USFhEkebi5cmYxaK-1)

`KBreadcrumbs` component reskin

Links:

- [Kongponents sandbox](https://deploy-preview-2051--kongponents-sandbox.netlify.app/#/breadcrumbs)
- [Kongponents docs preview](https://deploy-preview-2051--kongponents.netlify.app/components/breadcrumbs.html)

## PR Checklist

* [ ] **Conventional Commits** all commits follow the conventional commit standards [outlined in the main README](https://github.com/Kong/kongponents#committing-changes).
* [ ] **Tests coverage:** test coverage was added for new features and bug fixes
* [ ] **Docs:** includes a technically accurate README

### CODE REVIEW

**Comment by adamdehaven** [comment]
image
- The icon here looks a bit large?
- Also seems like more space is needed to the right of the icon, before the first item?

image
Why can I hover/click on the first two items, but no hover state on the third "You are not..." item?

**Comment by adamdehaven** [comment]
image

I wouldn't think I could hover/click on the last, current item? Wouldn't that be the page I'm already on, so it should have no action?

File: docs/components/breadcrumbs.md [hunk 1]
```diff
@@ -1,299 +1,255 @@
...
-  emphasis
 />
 ```
 
 ## Slots
 
 ### divider
 
-Content to be displayed between breadcrumb items, defaults to a chevron.
+Content to be displayed between breadcrumb items, defaults to forward slash.
```
Comment (adamdehaven) [inline docs/components/breadcrumbs.md:106]:
```suggestion
Content to be displayed between breadcrumb items, defaults to a forward slash `/`.
```

File: docs/components/breadcrumbs.md [hunk 2]
```diff
@@ -1,299 +1,255 @@
...
 ```
 
-#### Breadcrumb content
-
-Each breadcrumb item can display `text`, an `icon`, or both.
-
-#### Breadcrumb link
-
-The `to` property can be a Vue route or traditional URL. When using a URL though the target will be blank and a new window will open. In most scenarios you will build your breadcrumb items using your Vue application routes.
+The `to` property can be a Vue router object or a URL. When using a URL though the target will be blank and a new window will open. In most scenarios you will build your breadcrumb items using your Vue application routes.
```
[the remark this answers is not in the document]
Comment (portikM) [reply docs/components/breadcrumbs.md:62]: Good point, removing the `target: '_blank'` on links

**Comment by portikM** [comment]
> image
> 
> * The icon here looks a bit large?
> * Also seems like more space is needed to the right of the icon, before the first item?
> 
> image Why can I hover/click on the first two items, but no hover state on the third "You are not..." item?

Yeah the old example was weird - that was supposed to be a non-link item but there was nothing to indicate that. Updated now.

**Comment by portikM** [comment]
> image
> 
> I wouldn't think I could hover/click on the last, current item? Wouldn't that be the page I'm already on, so it should have no action?

I guess this makes sense - update component so that the active (last) item is not a link.

**Review by adamdehaven — APPROVED** [review]
lgtm

**Pull request merged**